src/squlearn/kernel/matrix/fidelity_kernel.py

```diff
@@ -80,8 +80,8 @@ def __init__(
     ) -> None:
         super().__init__(feature_map, executor, initial_parameters, parameter_seed, regularization)
 
-        self._quantum_kernel = None
-        self._evaluate_duplicates = evaluate_duplicates.lower()
+        self._quantum_kernel = None  # why .lower() anyways?
+        self._evaluate_duplicates = evaluate_duplicates  # .lower()
```

Comment:
let's remove these unnecessary comments.
```suggestion
        self._quantum_kernel = None
        self._evaluate_duplicates = evaluate_duplicates
```